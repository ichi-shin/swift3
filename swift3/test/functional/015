#!/bin/bash

# PUT Bucket versioning

. ./common

S3USER=tester

echo '<VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"> 
<Status>Enabled</Status>
</VersioningConfiguration>' > $tmp.version.enabled.xml

echo '<VersioningConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"> 
<Status>Suspended</Status>
</VersioningConfiguration>' > $tmp.version.suspended.xml

_s3_put /bucket
_s3_put /bucket?versioning -T $tmp.version.enabled.xml

_s3_put /bucket?versioning -T $tmp.version.suspended.xml -D - | _filter_curl
