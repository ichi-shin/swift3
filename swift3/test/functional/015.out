QA output created by 015
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -T /TMP.version.enabled.xml http://SWIFT_HOST/bucket?versioning... 200
> s3curl --id tester -- -X PUT -T /TMP.version.suspended.xml -D - http://SWIFT_HOST/bucket?versioning... 200
HTTP/1.1 100 Continue

HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

